pulsar-io/debezium/core/src/main/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistory.java

```diff
@@ -100,6 +100,8 @@ public final class PulsarDatabaseHistory extends AbstractDatabaseHistory {
         DatabaseHistory.NAME,
         READER_CONFIG);
 
+    protected static final String DURABLE_SUB_NAME = "durable-sub";
```

Comment:
It might also confuse user if they set a specific data retention.
Is it better to use retention policy.

For example, if the topic is not created, the connector can create the topic with infinite data retention by default.

Reply:
Under the current functionality, this topic must have infinite retention. Allowing users to set a custom data retention policy could lead to incomplete data and recovery failures.

Furthermore, if we were to manage this using data retention policies, it would introduce a dependency on pulsar-admin and likely change the permission requirements for the user when creating a connector.

Therefore, using a durable subscription is a simple and effective solution.

Reply:
@shibd If message TTL and retention are already configured on the namespace level, your change is meaningless. To achieve infinite retention, it's recommended to use pulsar-admin and set the retention value to 0, which disables retention.